.gitattributes

```diff
@@ -573,7 +573,6 @@ tools/h5diff/testfiles/tmpSingleSiteBethe.output.h5 -text
 tools/h5diff/testfiles/tmpSingleSiteBethe.reference.h5 -text
 tools/h5diff/testfiles/tmptest.he5 -text
 tools/h5diff/testfiles/tmptest2.he5 -text
-tools/h5dump/errfiles/non_existing.err -text
```

Comment:
Not related to this PR specifically, but shouldn't all these filenames be of the form `tools/test/${tool}/...`? Looks like maybe the paths weren't updated after tools refactoring?

Reply:
Looks like that needs fixed in develop!